dts/msm8916/msm8916-samsung-r08.dts

```diff
@@ -16,6 +16,19 @@
 		lk2nd,match-bootloader = "A500F*";
 	};
 
+	a53g {
+		model = "Samsung Galaxy A5 (SM-A500H)";
+		compatible = "samsung,a53g", "qcom,msm8916", "lk2nd,device";
+		lk2nd,match-bootloader = "A500H*";
+		lk2nd,smd-rpm-hack-opening;
```

Comment:
Are you sure that you need this? Can you test if the device works without the rpm-hack? I'd like to avoid enabling this without a reason. (make sure to drop it in both nodes)